packages/marshal/test/test-marshal-capdata.js

```diff
@@ -185,7 +184,41 @@ test('unserialize extended errors', t => {
   t.false('extraProp' in unkErr);
   t.false('cause' in unkErr);
   t.false('errors' in unkErr);
-  console.log('error with extra prop', unkErr);
+});
+
+test('unserialize errors w recognized extensions', t => {
+  if (typeof AggregateError === 'undefined') {
+    t.pass('skip test on platforms prior to AggregateError');
+    return;
+  }
```

Comment:
I think conditionally calling `test` vs. `test.skip` would produce clearer output (here and in the other test files):
```suggestion
const testIfAggregateError = typeof AggregateError !== 'undefined'
  ? test
  : test.skip;

testIfAggregateError('unserialize errors w recognized extensions', t => {
```